web/src/components/UserDropdown.tsx

```diff
@@ -100,44 +127,31 @@ export function UserDropdown({
                 overscroll-contain
               `}
           >
-            {showAdminPanel && (
-              <>
-                <Link
+            {showAdminPanel ? (
+              <DropdownOption
+                href="/admin/indexing/status"
+                icon={<LightSettingsIcon className="h-5 w-5 my-auto mr-2" />}
+                label="Admin Panel"
+              />
+            ) : (
+              showCuratorPanel && (
+                <DropdownOption
                   href="/admin/indexing/status"
-                  className="flex py-3 px-4 cursor-pointer !
-                   rounded hover:bg-hover-light"
-                >
-                  <LightSettingsIcon className="h-5 w-5 my-auto mr-2" />
-                  Admin Panel
-                </Link>
-              </>
-            )}
-            {showCuratorPanel && (
-              <>
-                <Link
-                  href="/admin/indexing/status"
-                  className="flex py-3 px-4 cursor-pointer !
-                   rounded hover:bg-hover-light"
-                >
-                  <LightSettingsIcon className="h-5 w-5 my-auto mr-2" />
-                  Curator Panel
-                </Link>
-              </>
+                  icon={<LightSettingsIcon className="h-5 w-5 my-auto mr-2" />}
+                  label="Curator Panel"
+                />
+              )
             )}
```

Comment:
fixed logical issue that would cause the border to show when only logout was present 

also allows admins/curators to logout from admin/curator menu